core/compilers.ts

```diff
@@ -1,10 +1,23 @@
+import { load as loadYaml, YAMLException } from "js-yaml";
+
 import * as utils from "df/core/utils";
 import { SyntaxTreeNode, SyntaxTreeNodeType } from "df/sqlx/lexer";
 
-export function compile(code: string, path: string) {
+export function compile(code: string, path: string): string {
   if (path.endsWith(".sqlx")) {
     return compileSqlx(SyntaxTreeNode.create(code), path);
   }
+  if (path.endsWith(".yaml")) {
+    try {
+      const yamlAsJson = JSON.stringify(loadYaml(code) as { [key: string]: number });
+      return `exports.asJson = () => (${yamlAsJson})`;
```

Comment:
why bother with the function? why not return the JSON string directly?

Reply:
NodeVM wraps it in a bunch of boilerplate, which makes that not work due to the wrapper

![image](https://github.com/dataform-co/dataform/assets/13434377/4ba23478-83c8-476e-af0b-80785b6bd527)

Putting an exports seemed like the clean way to do it

Reply:
Fair enough!